features/tabelwaarden.feature

```diff
@@ -0,0 +1,90 @@
+Functionaliteit: Als developer wil ik waardetabellen gebruiken in plaats van enumeraties
+  zodat ik mijn code niet hoef aan te passen als een enumeratielijst wordt aangepast die ik in mijn functionaliteit niet gebruik.
+
+  Wanneer in de registratie specifieke waarden in een enumeratie voorkomen, worden deze waarden in de API doorgegeven middels een waardetabel.
+  Wanneer in de registratie een omschrijving gekoppeld is aan een specifieke enumeratiewaarde, wordt deze omschrijving raadpleegbaar gemaakt middels de waardetabellen API.
+
+
+  Voor de enumeratie AanduidingBijzonderNederlanderschap wordt de tabel met tabelidentificatie "aanduidingbijzondernederlanderschap" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon behandeld wordt als Nederlander, of dat door de rechter is vastgesteld dat de persoon niet de Nederlandse nationaliteit bezit."
+  In de tabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | B                                                | Behandeld als nederlander        |
+    | V                                                | Vastgesteld Niet-Nederlander     |
+
+  Voor de enumeratie AanduidingBijHuisnummer wordt de tabel met identificatie "aanduidingbijhuisnummer" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "De aanduiding die wordt gebruikt voor adressen die geen straatnaam en huisnummeraanduidingen hebben."
+  In de waardetabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | to                                               | tegenover                        |
+    | by                                               | bij                              |
+
+  Voor de enumeratie Geslacht wordt de tabel met identificatie "geslacht" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon een man of vrouw is, of dat het geslacht (nog) onbekend is."
```

Comment:
hier noem je in de omschrijving van de tabel al de mogelijke waarden. dan kan je alsnog niet uitbreiden, want daarmee zou de betekenis van de tabel wijzigen.

Reply:
Ik heb wat te makkelijk de huidige enumeratie descriptions overgenomen. Dit is inderdaad geen goede tabel-omschrijving.

Reply:
Aangepast